target/ppc/translate.c

```diff
@@ -3196,6 +3196,21 @@ static inline void gen_align_no_le(DisasContext *ctx)
                       (ctx->opcode & 0x03FF0000) | POWERPC_EXCP_ALIGN_LE);
 }
 
+static inline bool do_ea_calc(DisasContext *ctx, int ra, TCGv displ, TCGv ea)
+{
+
+    if (ra) {
+        tcg_gen_add_tl(ea, cpu_gpr[ra], displ);
+    } else {
+        tcg_gen_mov_tl(ea, displ);
+    }
+    if (NARROW_MODE(ctx)) {
+        tcg_gen_ext32u_tl(ea, ea);
+    }
+
+    return true;
```

Comment:
Since we're always returning true, I assume we're not using the return value from do_ea_calc() anywhere, right? If so, drop the return and make it a void function.